frontend/javascripts/test/model/binary/cube.spec.js

```diff
@@ -238,16 +238,3 @@ test("Garbage Collection should not collect buckets with shouldCollect() == fals
   const addresses = cube.buckets.map(b => b.zoomedAddress);
   t.deepEqual(addresses, [[0, 0, 0, 0], [3, 3, 3, 0], [2, 2, 2, 0]]);
 });
-
-test("Garbage Collection should not throw an exception if no bucket is collectable", t => {
```

Comment:
I removed this test, since I find my new test (which provokes the error) more important than this one. For keeping both, I didn't have a good idea, and I also think that it's not super valuable.